Tribler/community/tunnel/main.py

```diff
@@ -304,7 +305,7 @@ def cb(ds):
             # Introduce dispersy port from other main peer to this peer
             line_split = line.split(' ')
             to_introduce = int(line_split[1])
-            self.anon_tunnel.community.add_discovered_candidate(Candidate(('127.0.0.1', to_introduce), tunnel=False))
+            self.anon_tunnel.community.add_discovered_candidate(Candidate((self.anon_tunnel.dispersy._wan_address[0], to_introduce), tunnel=False))
```

Comment:
I think this is wrong, Dispersy should be listening on 0.0.0.0. Hence also on 127.0.0.1. The wan_address is guessed, so should be incorrect more often.